geode-core/src/main/java/org/apache/geode/admin/ThreadMonitoring.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.admin;
+
+import java.util.Map;
+
+import org.apache.geode.internal.statistics.AbstractExecutorGroup;
+
+public interface ThreadMonitoring {
+
+  public enum Mode {
+    FunctionEx, PooledEx, SerialQueuedEx, OneTaskOnlyEx, ScheduledThreadEx, AGSEx
```

Comment:
I'm assuming the suffix "Ex" stands for Executor. I recommend either removing it or spelling it out as "Executor"

Reply:
all changes that u required were implemented and commited yesterday
For some reason the CI that was triggered after I committed my changes failed
This symbol is well defined under – ConfigurationProperties
Also I did not make any changes there since my last commit in which CI was successful
Locally CI is working fine
is it possible that for some reason during the CI - ConfigurationProperties.java that was used is not the one that i commited 14 days ago with those required changes?